property.go

```diff
@@ -135,73 +135,144 @@ type Option struct {
 	Color Color      `json:"color,omitempty"`
 }
 
-type DateProperty struct {
+type DatabaseDateProperty struct {
 	ID   ObjectID     `json:"id,omitempty"`
 	Type PropertyType `json:"type"`
-	Date interface{}  `json:"date"`
+	Date struct{}     `json:"date"`
 }
 
-func (p DateProperty) GetType() PropertyType {
+func (p DatabaseDateProperty) GetType() PropertyType {
 	return p.Type
 }
 
-type PeopleProperty struct {
+type PageDateProperty struct {
+	ID   ObjectID     `json:"id,omitempty"`
+	Type PropertyType `json:"type"`
+	Date Date         `json:"date"`
+}
+
+func (p PageDateProperty) GetType() PropertyType {
+	return p.Type
+}
+
+type DatabasePeopleProperty struct {
 	ID     ObjectID     `json:"id,omitempty"`
 	Type   PropertyType `json:"type"`
-	People interface{}  `json:"people"`
+	People struct{}     `json:"people"`
 }
 
-func (p PeopleProperty) GetType() PropertyType {
+func (p DatabasePeopleProperty) GetType() PropertyType {
 	return p.Type
 }
 
-type FileProperty struct {
-	ID   ObjectID     `json:"id,omitempty"`
-	Type PropertyType `json:"type"`
-	File interface{}  `json:"file"`
+//TODO: is this a right property format?
```

Comment:
Yes, for those that have been added in this change 👍 

A few others have been left out: (perhaps to be added in an ulterior commit?)

- The `PageRelationProperty` is missing (we only have `RelationProperty` at the moment, which should be renamed to `DatabaseRelationProperty`)
- Same applies to  `RichTextProperty`, `RollupProperty`, `FormulaProperty`, `CreatedByProperty`, `CreatedTimeProperty`,`LastEditedByProperty`